config.go

```diff
@@ -371,6 +373,21 @@ func loadConfig() (*config, []string, error) {
 		os.Exit(0)
 	}
 
+
+	// Update the home directory for dcrd if specified. Since the home
+	// directory is updated, other variables need to be updated to
+	// reflect the new changes.
+	if len(preCfg.DcrdHomeDir) > 0 {
+		cfg.DcrdHomeDir, _ = filepath.Abs(preCfg.DcrdHomeDir)
+		cfg.ConfigFile     = filepath.Join(cfg.DcrdHomeDir, defaultConfigFilename)
+		cfg.DataDir        = filepath.Join(cfg.DcrdHomeDir, defaultDataDirname)
+		cfg.RPCKey         = filepath.Join(cfg.DcrdHomeDir, "rpc.key")
+		cfg.RPCCert        = filepath.Join(cfg.DcrdHomeDir, "rpc.cert")
+		cfg.LogDir         = filepath.Join(cfg.DcrdHomeDir, defaultLogDirname)
+		fmt.Println("Using Home Directory: ", cfg.DcrdHomeDir)
```

Comment:
This shouldn't be printed without using the log, otherwise it won't match or go to the log file.  However, the logging isn't initialized yet.  So, this statement either needs to moved down after the logging is initialized, or removed.